Assets/Scripts/Device/DiscordControls.cs

```diff
@@ -42,28 +42,28 @@ public class DiscordControls {
         if (details != "") {
```

Comment:
In C#, if a block has one action inside it, you can usually remove the curly braces altogether.
I'm only making one comment about it, but I hope you update all occurences of this problem.
```suggestion
        if (details != "") rpDetais = details;
```